### PR TITLE
abdalla/UNIFY: tabs effect

### PR DESCRIPTION
**Pull request by AbdallaMourad**
## Changes Affect the following pages:

- [x] about
- [x] markets
- [x] terms-and condition
- [x] help-centre

### CODE REVIEW

**Review by kevinw-binary — DISMISSED** [review]

File: src/components/hooks/use-tab-state.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { useState, useEffect } from 'react'
+import { checkElemInArray, getLocationHash, isBrowser, routeBack, scrollTop, setLocationHash,  } from 'common/utility'
+
+export const useTabState = (tab_list) => {
+    const [active_tab, setActiveTab] = useState(getLocationHash() ? getLocationHash() : tab_list[0])
```
Comment (mustofa-binary) [inline src/components/hooks/use-tab-state.js:5]: we only rely on hash when its mounted, i guess we can add getlocation hash inside didmount useEffect. and then listen to active_tab to set location hash scrolltop, or routeback

File: src/pages/about/index.js [hunk 2]
```diff
@@ -157,27 +156,20 @@ const StyledText = styled(Text)`
...
-    const [active_tab, setTab] = useTabState('story')
-    const is_story = active_tab === 'story'
-    const is_leadership = active_tab === 'leadership'
+    const [active_tab, setActiveTab] = useTabState(['story', 'leadership'])
+    const [is_story, setStory] = useState(false)
+    const [is_leadership, setLeadership] = useState(false)
+
+    useEffect(() => {
+        setMobile(isBrowser() && getWindowWidth() <= size.tablet)
+    }, [getWindowWidth()])
```
Comment (mustofa-binary) [inline src/pages/about/index.js:167]: is this working as expected?

Comment (AbdallaMourad) [reply src/pages/about/index.js:167]: Yes it is

File: src/components/hooks/use-tab-state.js (hunk 1 above)
Comment (AbdallaMourad) [reply src/components/hooks/use-tab-state.js]: This sets the location if the user intendedly enter the URL with the hash provided.

**Review by abdalla-binary — APPROVED** [review]

**Review by mustofa-binary — APPROVED** [review]

**Pull request merged**